dracut-saltboot/saltboot/saltboot-root.sh

```diff
@@ -2,10 +2,13 @@
 
 if [ -z "$root" ] ; then
   root=saltboot
+else
+  # we have root, we can eventually continue without network after timeout
+  # the timeout is controlled by rd.retry option
+  echo "rm -f $hookdir/initqueue/finished/wait-network.sh" > $hookdir/initqueue/timeout/saltboot-network.sh
```

Comment:
Are you sure that `$hookdir` will not have space somewhere in it? Can it be specified manually by user? I would prefer it to be escaped.

Reply:
it is hardcoded in dracut-lib.sh:
hookdir=/lib/dracut/hooks
It is used everywhere in dracut without quotes.

Reply:
True, but at least they use `rm -f -- $hookdir...` to prevent passing unwanted arguments.